react/ProductSummaryContext.tsx

```diff
@@ -150,27 +171,15 @@ function ProductSummaryProvider({ product, selectedItem, children }) {
 }
 
 function useProductSummaryDispatch() {
-  const context = useContext(ProductDispatchContext)
-
-  if (context === undefined) {
-    throw new Error('useProductSummaryDispatch must be used within a ProductSummaryDispatchProvider')
-  }
-
```

Comment:
why did you remove this?

Reply:
Some components may be in both PDP or product-summary. If one of those components uses this context, it will work on product-summary but break on PDP.